page-home.php

```diff
@@ -12,8 +12,16 @@
 <!-- end main-content -->
 </main>
 
-<nav id="home-plus">
-  <a href="<?php echo site_url('/noticias'); ?>">+</a>
+<nav class="home-plus only-desktop">
+  <div class="container">
```

Comment:
why does this need `.container .row. col`?
IMO it should be as simple as `nav a` with some utility classes
Also this markup is overlapping the content in`.main-content`

![](https://puu.sh/r8USR/e1d9e8fb01.png)